roles/deploy/hooks/build-after.yml

```
@@ -12,14 +12,13 @@
 - name: Setup composer authentications
   composer:
     command: config
-    arguments: --auth http-basic.{{ composer_authentication.hostname }} {{ composer_authentication.username }} {{ composer_authentication.password }}
+    arguments: --auth http-basic.{{ composer_authentication.hostname }} {{ composer_authentication.username }} {{ composer_authentication.password | default("") | quote }}
```

```suggestion
    arguments: --auth http-basic.{{ composer_authentication.hostname | quote }} {{ composer_authentication.username | quote }} {{ composer_authentication.password | default("") | quote }}
```

Although `hostname` should never contains whitespaces, quoting it anyway for security.